pkg/webui/console/views/device-general-settings/identity-server-form/index.js

```diff
@@ -38,8 +38,7 @@ import validationSchema from './validation-schema'
 
 const messages = defineMessages({
   deleteDevice: 'Delete end device',
-  deleteWarning:
-    'Are you sure you want to delete "{deviceId}"? This action cannot be undone and it will not be possible to reuse the end device ID.',
+  deleteWarning: 'Are you sure you want to delete "{deviceId}"?',
```

Comment:
```suggestion
  deleteWarning: 'Are you sure you want to delete "{deviceId}"? This action cannot be undone and the end device ID will not be immediately reusable.',
```
I think it's still good to make users aware that the IDs are typically not reusable right away but after a retention time only.

cc @htdvisser

Reply:
See https://www.thethingsindustries.com/docs/reference/id-eui-constraints/#deleted-entities

> When an end device is deleted, the ID and EUI are released and can be reused

Reply:
I see! See my updated change suggestion then @asmulko